CHANGELOG.md

```diff
@@ -0,0 +1,72 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## 0.0.1 - 2018-10-31
+### Added
+- Add PPO and DDPG for TensorFlow branch
+- Add HER to DDPG
+- Add recurrent policy to NPO
+- Base class for ReplayBuffer, and two implementations: SimpleReplayBuffer
+  and HerReplayBuffer
+- Sampler classes OffPolicyVectorizedSampler and OnPolicyVectorizedSampler
+- Base class for offline policies OffPolicyRLAlgorithm
+- Benchmark tests for TRPO, PPO and DDPG to compare their performance with
+  those produced by OpenAI baselines
+- Dynamics randomization for MuJoCo environments
+- Support for dm_control environments
+- Add dict support for garage environments
+- PEP8 checks enforced in the codebase
+- Support to keep all imports in python files ordered, removed if they're
+  not required, as well as free of import errors.
+- TravisCI support using Docker
+- Test tree under for all the code developed in garage.
+- Code Coverage measurement with codecov
+- Git hooks to enforce PEP8, and to verify imports and commit messages. They're
+  also applied in the Travis CI verification
+- Docstring verification for added files that are not in the test branch or
+  moved files
+- TensorBoard Support
+- Variable and name scope for symbolic operations in TensorFlow
+- Add top-level base Space class for garage
+- Asynchronous plotting for Theano and Tensorflow
+
+
+### Changed
+- Rename rllab to garage, including all the rllab references in the packages
+  and modules inside the project
+- Rename run_experiment_lite to run_experiment
+- The file cma_es_lib.py was replaced by the pycma library available on PyPI
+- Move the contrib package to garage.contrib
+- Move Theano-dependent code in garage.theano
+- Move all code from sandbox.rocky.tf to garage.tf
+- Update several dependencies, mainly:
+  - python to 3.6.6
+  - TensorFlow to 1.9
+  - Theano to 1.0.2
+  - mujoco-py to 1.50.1
+  - gym to 0.10.8
+- Various dependencies were moved from conda to be installed with pip
+- Separate example script files in the Theano and TensorFlow branch
+- Update LICENSE, CONTRIBUTING.md and .gitignore
+- Use convenience imports, that is, import classes and functions that share the
+  same or similar name to its module in the corresponding __init__.py file of
+  their package
+- Replace ProxyEnv with gym.Wrapper
+- Update installation scripts for Linux and OSX
```

Comment:
Let's consider replacing OSX with macOS as that is now the official name Apple is going with for their OS.